.github/workflows/chromatic.yml

```diff
@@ -0,0 +1,19 @@
+name: "Chromatic"
+on: push
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v1
+      - name: Use Node.js
+        uses: actions/setup-node@v1
+        with:
+          node-version: "10.16.3"
+      - name: Install dependencies
+        run: yarn setup
```

Comment:
Note: this crucial step (yarn or npm) is not mentioned in the Chromatic github action readme (https://github.com/chromaui/action). I got most stuck on this, as the error message is very confusing if you don't do this! I will get them to update their docs.